### PR TITLE
Translations update from Weblate

### PR DESCRIPTION
**Pull request by weblate-fpf**
Translations update from [Weblate](https://weblate.securedrop.org) for [SecureDrop/SecureDrop Client](https://weblate.securedrop.org/projects/securedrop/securedrop-client/).

Current translation status:

Weblate translation status

### CODE REVIEW

**Comment by cfm** [comment]
Per <https://github.com/freedomofpress/securedrop-client/pull/1516#issuecomment-1163447454>, note that this diff still shows some flapping newlines, because this pull request doesn't contain any new commits from Weblate's `.po` exporter.  I think the path of least resistance is to let the flapping resolve in subsequent commits, whether in this or a future pull request from Weblate.

**Review by gonzalo-bulnes — CHANGES_REQUESTED** [review]
There are mis-translations that seem to be caused by Weblate itself (by that I mean that I don't think they were caused by the translators). I'm marking this as blocking so that we can discuss it :slightly_smiling_face:

File: securedrop_client/locale/hr/LC_MESSAGES/messages.po
Comment (cfm) [reply securedrop_client/locale/hr/LC_MESSAGES/messages.po:81]: This looks consistent with gettext's expected production of ["fuzzy" entries](https://www.gnu.org/software/gettext/manual/html_node/Fuzzy-Entries.html) based on existing translations.  That's a feature, implicitly on the expectation that reviewing and (probably) revising these translations is still less work than generating them from scratch, if even some can be approved as is.

This is the same behavior we see [in `securedrop`](https://github.com/freedomofpress/securedrop/search?l=Gettext+Catalog&q=fuzzy) insofar as we *don't* use `msgmerge --no-fuzzy-matching` or its Babel equivalent.  I'd be open to discussing that as a procedural change in how we propagate source-string changes for translation, but in the meantime I think it should be entirely *un*controversial for translators to correct these entries in the course of translation and approval!

Comment (gonzalo-bulnes) [reply securedrop_client/locale/hr/LC_MESSAGES/messages.po:81]: Yeah, I think those automation features may require a hit-miss-ratio analysis. (I would involve the people who are translating, but also ourselves. Fro example, I certainly think it makes review less straightforward y introducing noise that's not easy to distinguish from the signal.) I don't think we're in a rush to make a decision on that, what do you think @cfm, when would be a good time to reaching out to folks?

Comment (cfm) [reply securedrop_client/locale/hr/LC_MESSAGES/messages.po:81]: @gonzalo-bulnes, I would probably defer to Localization Lab.  I'll file a ticket to reconsider this across `securedrop[-client]` for Server 2.5.0, which is our next scheduled point of coordination with them.

Comment (gonzalo-bulnes) [reply securedrop_client/locale/hr/LC_MESSAGES/messages.po:81]: What does this mean for this PR? Are these automated updates something we want to get incrementally at all? What are the advantages/drawbacks to that?

**Comment by gonzalo-bulnes** [comment]
@cfm and I had an offline chat about 

**Context**

1. Only the strings belonging to an **enabled** language are packaged with new releases
2. Weblate will create a PR and keep it updated:
  - every time a string changes
  - making one commit per language
  - ensuring that if new changes come in, the commit always reflect them (on an hourly basis)

**Decisions**

- Because  only the strings belonging to an **enabled** language are packaged with new releases (1):
  - Incorrect translations in non-enabled languages are therefore not concerning.
  - We're comfortable integrating non-enabled languages on a regular basis.
  
- Because **enabled** languages can be used by journalists after each release:
  - Enabled languages need to be reviewed for **correctness** before each release (more on that below)
  - :cherries: As long as that review is performed before release, we-re comfortable integrating enabled languages on a regular basis as well.
  - :bulb: If the level of support for any given language ever became insufficient to ensure correct translations are shipped for each release, we can _disable_ support for that language.
  
**Reviewing strings**

Translated strings need to be reviewed for:

- Validity: to make sure they don't break the localization system. (Weblate is unlikely to produce invalid strings / PO files.)
- Correctness: to make sure they're accurate, with the following in mind:
  1. Inaccurate strings can be unsafe (e.g. HTML/JS injections, misleading instructions)
  2. Inaccurate strings can reduce accessibility (e.g. confusing grammar or terminology)
  - There can be overlap between both concerns (e.g. people can make dangerous mistakes if things are unintentionally unclear)
  
:cherries: The safety aspect of the review happens in GitHub, either when Weblate opens a PR or _at the very least_ before each release. The accessibility aspect of the review can happen in Weblate, in conversation with the localization team, but MUST be taken into account before each release to evaluate the level of support that each **enabled** language actually has.

----

**Weblate/GitHub flow**

- Review and merge Weblate PRs whenever convenient, Weblate will keep its PRs updated, so there is usually no rush.
- :fire: Before each release, perform a thorough _correctness_ review of all changes affecting supported languages.

----

**Longer term vision**

Localization is an integral part of building accessible user interfaces. We recognize that there is a collaboration opportunity between engineering/design/research (FPF) and the translation/localization team. This is a first step towards making it possible for translations to happen at any time during the release process, instead of only within a few days prior to each release.

We take this step as an incremental step towards a more collaborative integration with the translation/localization team, and aim at tightening those ties over time. :green_heart: 

Examples of what we'd like to achieve through closer collaboration:

- Making sure translators have the clear context they need to understand the intention behind each copy decision. (e.g. Was "Reply" a noun or a verb in this context?)
- Making sure the original copy (English) is refined to make as many translations as possible unambiguous / easier. (e.g. distinguishing different concepts that may originally be covered by the same word in English, if that makes the copy clearer and the translation less error-prone.)
- Improving our ability to identify design changes to improve how the user interface can be localized effectively: e.g. buttons behavior when rendered in a RTL/LTR/mixed context.

**Comment by gonzalo-bulnes** [comment]
With the [above](https://github.com/freedomofpress/securedrop-client/pull/1527#issuecomment-1175604218) in mind, I'll take a fresh look at this PR and merge it.

(We also said it would be nice for all languages to use the commit message template that includes the language name, but that's not a blocker.)

**Comment by cfm** [comment]
Thanks for summarizing, @gonzalo-bulnes!  I think everything you've outlined is consistent with the current [overview](https://github.com/freedomofpress/securedrop-client/wiki/Internationalization#overview), but feel free to make (or suggest) any changes there too.

**Comment by gonzalo-bulnes** [comment]
I'd outline an explicit responsibility for the release team to verify the quality of the translations as much as possible. (Because the strings come otherwise from an external group of people, and we're the last opportunity to check them before they make their way to the journalists' workstations.) Let's think of the phrasing together. (And thanks for adding the reference here @cfm, it's great context!)

**Comment by creviera** [comment]
Since translations are all disabled by default and MANIFEST.in doesn't have any included languages yet, this seems fine to pull into this upcoming release

**Review by gonzalo-bulnes — APPROVED** [review]
Oops, I forgot documenting my approving review!
I'm merging now :slightly_smiling_face:

**Pull request merged**
